app/views/api/v2/discovered_hosts/main.json.rabl

```diff
@@ -2,7 +2,8 @@ object @discovered_host
 
 extends "api/v2/discovered_hosts/base"
 
-attributes :ip, :mac, :last_report, :subnet_id, :subnet_name, :cpu, :memory, :disk_count, :disks_size
+attributes :ip, :mac, :last_report, :subnet_id, :subnet_name, :memory, :disk_count, :disks_size
+attribute :cpu_count => :CPUs
```

Comment:
we show it with capital letters in the UI and I couldn't think of a better way to differ the 's' but I can change it